packages/pds-ember/app/styles/pds/theme/_light.scss

```diff
@@ -202,12 +204,59 @@ $SHADOWS: (
 }
 
 
+///----------------------------------------///
+/// Headings
+///----------------------------------------///
+$heading-fontSize--0: scale.$lg--5; // 36px;
+$heading-fontSize--1: scale.$lg--3; // 28px
+$heading-fontSize--2: $fontSize--lg;
+$heading-fontSize--3: scale.$base; // 16px
+$heading-fontSize--4: 0.875rem; // 14px
```

Comment:
We could use a variable for the 14px font size, but I'm not sure what to call it. It's essentially a value between `scale.$base` and `scale.$sm--1`. However, it's difficult to modify the scale because it would require modifying several consuming modules (even externally) to update, which introduces risk of breaking existing styles (`theme.$size--*` variables have a similar problem).

Rather than define a variable for a value that's not a part of the modular scale, I'm using raw values instead.  If we figure out another strategy later on, we'll look into replacing them at that time.

Reply:
Will be defining a private `$_size14` variable to isolate usage of 14px font size to the theme file and provide an upgrade path if designs update in the future.